website/docs/r/hci_cluster.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_hci_cluster"
+description: |-
+  Manages an Azure Stack HCI Cluster.
+---
+
+# azurerm_hci_cluster
+
+Manages an Azure Stack HCI Cluster.
+
+## Example Usage
+
+```hcl
+data "azurerm_client_config" "current" {}
```

Comment:
this should likely be using the AAD Application Data Source to look up an existing application, rather than the service principal, which it's highly unlikely is being reused for AAD?

Reply:
Sorry.  For the other comments you mentioned, I've submitted PR. But for this comment, seems I have no permission to create an AAD application via "azuread_application". So I haven't updated test code against this case.

Reply:
OK. Updated test code.